src/assets/main.scss

```diff
@@ -51,6 +51,7 @@
   --pastel-blue: #216e93;
   --primary-blue: #0079ff;
   --secondary-blue: #0056b3;
+  --dark-blue: #0c1c27;
```

Comment:
New color provided by Baxter. The design team at some point will go over all types of blues that we have to see which ones are needed.